docs/bugs_and_glitches.md

```diff
@@ -175,24 +175,38 @@ This bug existed for all battles in Gold and Silver, and was only fixed for sing
 ```diff
 -	; BUG: 1/256 chance to fail even for a 100% effect chance,
 -	; since carry is not set if BattleRandom == [hl] == 255
+- 	call BattleRandom
 +	ld a, [hl]
-+	cp 100 percent
-+	jr z, .ok
- 	call BattleRandom
++	sub 100 percent
++	; If chance was 100%, RNG won't be called (carry not set)
++	; Thus chance will be subtracted from 0, guaranteeing a carry
++	call c, BattleRandom
  	cp [hl]
--	pop hl
--	ret c
-+	jr c, .ok
+	pop hl
+	ret c
 
  .failed
  	ld a, 1
  	ld [wEffectFailed], a
  	and a
-+.ok
-+	pop hl
  	ret
 ```
 
+If you wish to keep compatibility with standard Pokémon Crystal, you can disable the fix during link battles by also applying the following edit in the same place:
```

Comment:
This is something that is applicable to at least half of the bug fixes in this document. If you wish to document this, do it in a generic way, such that we do not have to repeat similar instructions for all of the bugs.

Reply:
That has already been done (see the top of the current document), but the problem is that it's really complicated for some of these functions which rely on flag behavior. SDH, who requested a review of their code, had trouble with exactly that, hence wanting to push a fix to master.
Further, following this reasoning, the entire document could be summarized as "the bugs can be solved by editing the code to remove the bugs", which wouldn't make sense.